### DISCUSSION TITLE
Do note links exist still?

### DISCUSSION THREAD

**Discussion by emcangi** [BoostIO/BoostNote.next-local#59] [Q&A] [answered]
In the original Boostnote legacy, you could right click to get a note link and paste that into other notes. I can't seem to find this anymore in Boostnote.next-local. Does it still exist? I did not see it on the Roadmap.

Thank you!

**Comment by Komediruzecki** [comment]
Hi,

the feature was actually implemented long ago but never merged, the code changed a bit from that time, so I need to refactor and merge it. 

It should ship in the next release.

**Reply by Komediruzecki** [reply]
The feature merged to master: https://github.com/BoostIO/BoostNote.next-local/pull/64

Will be available in the next release.

**Comment by Komediruzecki** [comment, marked as answer]
Available in latest release.
